libs/components/layout/src/lib/modules/back-to-top/back-to-top-adapter.service.ts

```diff
@@ -79,16 +79,14 @@ export class SkyBackToTopDomAdapterService implements OnDestroy {
   }
 
   public isElementScrolledInView(element: ElementRef): boolean {
-    const parentElement = this.scrollableHostService.getScrollableHost(
-      element.nativeElement
-    );
+    const parentElement = this.scrollableHostService.getScrollableHost(element);
```

Comment:
I'm still seeing some `private` variables in the `SkyBackToTopDomAdapterService`

Reply:
fixed